modules/core/src/test/scala/gem/config/Arbitraries.scala

```diff
@@ -40,57 +40,77 @@ trait Arbitraries extends gem.enum.Arbitraries {
   private def const[A](a: A): Arbitrary[A] =
     Arbitrary(Gen.const(a))
 
-  implicit val arbAcqCamStatic   = const(AcqCamStaticConfig()  )
-  implicit val arbBhrosStatic    = const(BhrosStaticConfig()   )
-  implicit val arbGmosNStatic    = const(GmosNStaticConfig()   )
-  implicit val arbGmosSStatic    = const(GmosSStaticConfig()   )
-  implicit val arbGnirsStatic    = const(GnirsStaticConfig()   )
-  implicit val arbGpiStatic      = const(GpiStaticConfig()     )
-  implicit val arbGsaoiStatic    = const(GsaoiStaticConfig()   )
-  implicit val arbMichelleStatic = const(MichelleStaticConfig())
-  implicit val arbNiciStatic     = const(NiciStaticConfig()    )
-  implicit val arbNifsStatic     = const(NifsStaticConfig()    )
-  implicit val arbNiriStatic     = const(NiriStaticConfig()    )
-  implicit val arbPhoenixStatic  = const(PhoenixStaticConfig() )
-  implicit val arbTrecsStatic    = const(TrecsStaticConfig()   )
-  implicit val arbVisitorStatic  = const(VisitorStaticConfig() )
-
-  implicit val arbF2Static       =
-    Arbitrary(arbitrary(arbBool).map(F2StaticConfig(_)))
+  implicit val arbMosPreImaging: Arbitrary[MosPreImaging] =
+    Arbitrary(
+      Gen.oneOf(MosPreImaging.IsMosPreImaging,
+                MosPreImaging.IsNotMosPreImaging)
+    )
+
+  implicit val arbAcqCamStatic    = const(AcqCamStaticConfig()         )
```

Comment:
Right, these are placeholders.  Having an arbitrary `AcqCamStaticConfig` instance available allows us to fill out the match in `genStaticConfigOf(i: Instrument)` and avoid a catch-all case or a missing case warning:

```scala
  def genStaticConfigOf(i: Instrument): Gen[StaticConfig] =
    i match {
      case AcqCam     => arbitrary[AcqCamStaticConfig   ].widen[StaticConfig]
      ...
    }
```